compiler/common-artifacts/CMakeLists.txt

```diff
@@ -24,13 +24,13 @@ else()
     find_package(Python 3.8 COMPONENTS Interpreter QUIET)
   endif()
 
-  # tensorflow 2.12.1 supports Python 3.8 ~ 3.11
-  if(Python_VERSION VERSION_GREATER_EQUAL 3.12)
-    message(STATUS "Build common-artifacts: FALSE (Python version 3.12 or higher is not supported yet)")
+  # tensorflow 2.12.1 supports Python 3.8 ~ 3.12
```

Comment:
this is false fix... 
I think I need to revise decision logic as we can't have single set of versions.

Reply:
I couldn't catch the point. Could you elucidate the problem?